### PR TITLE
Get badges displayed to shrink with smaller widget and font update

### PR DESCRIPTION
**Pull request by cbelan-d2l**
* Sizes to constants file for calculation purposes

### CODE REVIEW

File: src/components/award-issued.js [hunk 1]
```diff
@@ -23,8 +24,8 @@ class AwardIssued extends BaseMixin(LitElement) {
 				cursor: pointer;
 			}
 			.badgeEntry {
-				height: 35px;
-				width: 35px;
+				height: ${BadgeImageSize}px;
```
Comment (depowell-D2L) [inline src/components/award-issued.js]: You should be using `unsafeCss` for this. It help the lit html stuff make some safer choices. See https://github.com/Brightspace/google-calendar-widget-ui/blob/master/src/components/google-calendar-event.js#L43 for an example

File: src/components/leaderboard-row.js [hunk 2]
```diff
@@ -89,7 +90,7 @@ class LeaderboardRow extends BaseMixin(LitElement) {
 				margin-top: 11px;
 				margin-bottom: -20px;
 				transition: max-height 0.2s ease-out;
-				padding-left: 13px;
+				padding-left: ${PanelPadding}px;
```
Comment (depowell-D2L) [inline src/components/leaderboard-row.js]: Same here, use `unsafeCss`

File: src/components/leaderboard-row.js [hunk 3]
```diff
@@ -164,18 +162,25 @@ class LeaderboardRow extends BaseMixin(LitElement) {
...
+				<div class='d2l-body-standard noMargin displayNumber'>${this._getDisplayNumber()}</div>
 			`;
 		} else {
 			displayNumber = html`
 				<div class='d2l-body-small noMargin'>${this._getDisplayNumber()}</div>
 			`;
 		}
 
+		let fontStyle;
+		if (this._full) {
```
Comment (depowell-D2L) [inline src/components/leaderboard-row.js]: Why not make this a ternary? You could do it inline in the html template then

Comment (cbelan-d2l) [reply src/components/leaderboard-row.js]: Didn't realize ternary was available (until later on :) ) Thanks!
06a6e32

File: src/components/leaderboard-row.js (hunk 2 above)
Comment (cbelan-d2l) [reply src/components/leaderboard-row.js]: Fixed 06a6e32

File: src/components/award-issued.js (hunk 1 above)
Comment (cbelan-d2l) [reply src/components/award-issued.js]: Fixed 06a6e32
Thanks for that

**Pull request merged**